scripts/lib/mkcloud-driver-libvirt.sh

```diff
@@ -51,7 +51,7 @@ function libvirt_start_daemon()
 
 function libvirt_net_start()
 {
-    $sudo virsh net-start $cloud-admin || exit $?
+    $sudo virsh net-info $cloud-admin | grep 'Active.*yes' || $sudo virsh net-start $cloud-admin || exit $?
```

Comment:
Firstly, grep -q. Secondly, I'd much rather see this in an if block then jamming it onto one single line with two or conditions:

```
if ! $sudo virsh net-info $cloud-admin | grep -q 'Active.*yes' ; then
    $sudo virsh net-start $cloud-admin || exit $?
fi
```